client/src/main/java/tds/assessment/Item.java

```diff
@@ -137,10 +141,47 @@ public void setItemProperties(List<ItemProperty> itemProperties) {
     }
 
     /**
+     * @return The full path and file name of the item's XML file
+     */
+    public String getItemFilePath() {
+        return itemFilePath;
+    }
+
+    public void setItemFilePath(String itemFilePath) {
+        this.itemFilePath = itemFilePath;
+    }
+
+    /**
+     * @return The full path and file name of the item's stimulus file (if there is one)
+     */
+    public String getStimulusFilePath() {
+        return stimulusFilePath;
+    }
+
+    public void setStimulusFilePath(String stimulusFilePath) {
+        if (stimulusFilePath == null) {
```

Comment:
I agree.  For this particular case, here's how I understand the item/stimulus files:
* You can have an item that does not have associated stimuli
* You cannot have an item without an item XML file

I've added the null check to the `itemFilePath`, because that's what the database says.  As I'm looking through the legacy code for implementing get page content, I'll check to see if having a null `itemFilePath` is an exceptional case.  If it is, we can come back and refactor the getter/setter to act accordingly.  How's that sound?